docs/source/API/core/builtinreducers/Sum.rst

```diff
@@ -0,0 +1,99 @@
+``Sum``
+=======
+
+.. role::cpp(code)
+    :language: cpp
+
+.. role:: cppkokkos(code)
+    :language: cppkokkos
+
+Specific implementation of `ReducerConcept <ReducerConcept.html>`_ performing an ``add`` operation
+
+Header File: ``<Kokkos_Core.hpp>``
+
+Usage
+-----
+
+.. code-block:: cpp
+
+    T result;
+    parallel_reduce(N,Functor,Sum<T,S>(result));
+
+Synopsis 
+--------
+
+.. code-block:: cpp
+
+    template<class Scalar, class Space>
+    class Sum{
+        public:
+            typedef Sum reducer;
+            typedef typename std::remove_cv<Scalar>::type value_type;
+            typedef Kokkos::View<value_type, Space> result_view_type;
+            
+            KOKKOS_INLINE_FUNCTION
+            void join(value_type& dest, const value_type& src) const;
+
+            KOKKOS_INLINE_FUNCTION
+            void init( value_type& val) const;
+
+            KOKKOS_INLINE_FUNCTION
+            value_type& reference() const;
+
+            KOKKOS_INLINE_FUNCTION
+            result_view_type view() const;
+
+            KOKKOS_INLINE_FUNCTION
+            Sum(value_type& value_);
+
+            KOKKOS_INLINE_FUNCTION
+            Sum(const result_view_type& value_);
+    };
+
+Public Class Members
+--------------------
+
+Typedefs
+~~~~~~~~
+
+* ``reducer``: The self type.
+* ``value_type``: The reduction scalar type.
+* ``result_view_type``: A ``Kokkos::View`` referencing the reduction result 
+
+Constructors
+~~~~~~~~~~~~
+
+.. cppkokkos:kokkosinlinefunction:: Sum(value_type& value_);
+
+* Constructs a reducer which references a local variable as its result location.  
+
+.. cppkokkos:kokkosinlinefunction:: Sum(const result_view_type& value_);
+
+* Constructs a reducer which references a specific view as its result location.
+
+Functions
+~~~~~~~~~
+
+.. cppkokkos:kokkosinlinefunction:: void join(value_type& dest, const value_type& src) const;
+
+* Add ``src`` into ``dest``:  ``dest+=src;``. 
+
+.. cppkokkos:kokkosinlinefunction:: void init(value_type& val) const;
+
+* Initialize ``val`` using the Kokkos::reduction_identity<Scalar>::sum() method.  The default implementation sets ``val=0``.
+
+.. cppkokkos:kokkosinlinefunction:: value_type& reference() const;
+
+* Returns a reference to the result provided in class constructor.
+
+.. cppkokkos:kokkosinlinefunction:: result_view_type view() const;
+
+* Returns a view of the result place provided in class constructor.
```

Comment:
Word choice suggestion:  instead of "...result place provided in class constructor," try, "...result referenced in class constructor."

Reply:
Done!